controller/pgcluster/pgclustercontroller.go

```diff
@@ -314,3 +324,131 @@ func addIdentifier(clusterCopy *crv1.Pgcluster) {
 
 	clusterCopy.ObjectMeta.Labels[config.LABEL_PG_CLUSTER_IDENTIFIER] = string(u[:len(u)-1])
 }
+
+// updateTablespaces updates the PostgreSQL instance Deployments to reflect the
+// new PostgreSQL tablespaces that should be added
+func updateTablespaces(c *Controller, oldCluster *crv1.Pgcluster, newCluster *crv1.Pgcluster) error {
+	// first, get a list of all of the available deployments so we can properly
+	// mount the tablespace PVCs after we create them
+	// NOTE: this will also get the pgBackRest deployments, but we will filter
+	// these out later
+	selector := fmt.Sprintf("%s=%s,%s=%s", config.LABEL_VENDOR, config.LABEL_CRUNCHY,
+		config.LABEL_PG_CLUSTER, newCluster.Name)
+
+	deployments, err := kubeapi.GetDeployments(c.PgclusterClientset, selector, newCluster.Namespace)
+
+	if err != nil {
+		return err
+	}
+
+	// now get the instance names, which will make it easier to create all the
+	// PVCs
+	instanceNames := []string{}
+
+	for _, deployment := range deployments.Items {
+		labels := deployment.ObjectMeta.GetLabels()
+
+		// get the name of the PostgreSQL instance. If the "deployment-name"
+		// label is not present, then we know it's not a PostgreSQL cluster.
+		// Otherwise, the "deployment-name" label doubles as the name of the
+		// instance
+		if instanceName, ok := labels[config.LABEL_DEPLOYMENT_NAME]; ok {
```

Comment:
I don't think an issue here since I believe the deployment label is currently only on PG pods as stated here, but did want to mention that a `pgo-pg-database` label was added to the cluster deployment template to positively identify a pod or deployment that is a PG pod (i.e. a primary or replica), in the event that the more generic `deployment-name` label is used elsewhere.